packages/backend/src/routes/attestationStatsByAttester.mjs

```diff
@@ -0,0 +1,23 @@
+import catchError from '../helpers/catchError.mjs'
+
+export default ({ app, db, synchronizer }) => {
+  const handler = async (req, res) => {
+    const { attesterId } = req.params
+    const { data } = await db.findOne('AttesterData', {
+      where: {
+        _id: attesterId,
+      },
+    })
+    const stats = JSON.parse(data)
+    const attestationCount = await db.count('Attestation', {
+      //   where: {
+      //     attesterId: attesterId,
+      //   },
+    })
+    res.json({
+      ...stats,
+      attestationCount,
+    })
```

Comment:
I've got the new table added to accumulate each attester's attestations, and the new route is working except for this `where:`, I'm getting:
> Error: anondb error: Error: Unable to find row definition for key: "where"

should I be able to use `db.count` this way?

Reply:
Yes, but you don't need to specify a where clause.

So it's just

```js
await db.count('Attestation', {
  attesterId,
})
```